clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/ClusteredStore.java

```diff
@@ -249,66 +247,36 @@ public void clear() throws StoreAccessException {
       }
 
       // TODO: Create tiered configuration ala org.ehcache.impl.internal.store.tiering.CacheStore.Provider
-      final Store.Provider underlyingStoreProvider =
-          selectProvider(storeConfig.getResourcePools().getResourceTypeSet(), Arrays.asList(serviceConfigs));
-
-      final Store<K, V> underlyingStore = underlyingStoreProvider.createStore(storeConfig, serviceConfigs);
 
       ClusteredCacheIdentifier cacheId = findSingletonAmongst(ClusteredCacheIdentifier.class, (Object[]) serviceConfigs);
       ServerStoreProxy serverStoreProxy = clusteringService.getServerStoreProxy(cacheId, storeConfig);
       OperationCodecProvider<K, V> codecProvider =
           new OperationCodecProvider<K, V>(storeConfig.getKeySerializer(), storeConfig.getValueSerializer());
       OperationsCodec<K, V> codec = new OperationsCodec<K, V>(codecProvider);
       ChainResolver<K, V> resolver = new ChainResolver<K, V>(codec);
-      Store<K, V> store = new ClusteredStore<K, V>(serverStoreProxy, underlyingStore, codec, resolver);
+      Store<K, V> store = new ClusteredStore<K, V>(serverStoreProxy, codec, resolver);
 
-      createdStores.put(store, underlyingStoreProvider);
       return store;
     }
 
     @Override
     public void releaseStore(final Store<?, ?> resource) {
-      Store.Provider underlyingStoreProvider = createdStores.remove(resource);
-      if (underlyingStoreProvider == null) {
-        throw new IllegalArgumentException("Given store is not managed by this provider: " + resource);
-      }
-
       ClusteredStore clusteredStore = (ClusteredStore)resource;
-      underlyingStoreProvider.releaseStore(clusteredStore.underlyingStore);
       this.clusteringService.releaseServerStoreProxy(clusteredStore.storeProxy);
     }
 
     @Override
     public void initStore(final Store<?, ?> resource) {
-      Store.Provider underlyingStoreProvider = createdStores.get(resource);
-      if (underlyingStoreProvider == null) {
-        throw new IllegalArgumentException("Given store is not managed by this provider: " + resource);
-      }
-
-      underlyingStoreProvider.initStore(((ClusteredStore)resource).underlyingStore);
+      //no op
```

Comment:
Impossible - init cannot be a no-op since release will get rid of the `storeProxy`. Lifecycle needs to be preserved here.